<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addPlayer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportMatch">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Report Match</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findPlayer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addPlayerTag">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reportMatchTag1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findPlayerTag">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addPlayerScore">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reportMatchTag2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P1Wins">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>P1Wins</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P2Wins">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>P2Wins</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="importFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exportFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="display">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Bracket</string>
    </property>
   </widget>
   <widget class="QLabel" name="findPlayerOutput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>75</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
